lib/keypair.ex

```diff
@@ -36,24 +36,25 @@ defmodule Elixium.KeyPair do
   end
 
   @doc """
-    Using a public address, fetch the correct keyfile and return the only the private key
+    Creates a new mnemonic to give to users based off private key
   """
-  @spec get_priv_from_file(String.t()) :: {binary, binary}
-  def get_priv_from_file(pub) do
-    unix_address = Application.get_env(:elixium_core, :unix_key_address)
-    key_path = unix_address <> "/" <> pub <>".key"
-    {pub, priv} = get_from_file(key_path)
-    priv
+  @spec create_mnemonic(binary) :: String.t()
+  def create_mnemonic(private) do
+    Elixium.Advanced.from_entropy(private)
   end
 
-  @spec create_keyfile(tuple) :: :ok | {:error, any}
-  defp create_keyfile({public, private}) do
-    unix_address = Application.get_env(:elixium_core, :unix_key_address)
-    if !File.dir?(unix_address), do: File.mkdir(unix_address)
-
-    address = address_from_pubkey(public)
-
-    File.write(unix_address <> "/#{address}.key", private)
+  @doc """
+    Generates a keypair from the seed phrase
+  """
+  @spec gen_keypair(String.t()) :: {binary, binary}
+  def gen_keypair(phrase) do
+    case String.contains?(phrase, " ") do
```

Comment:
Whats the difference when it contains an empty space?

Reply:
empty space indicates its a mnemonic its importing from, will document